content/topics/gdpr-ethical-review-glossary.md

```diff
@@ -52,7 +52,7 @@ Data retention rules in the GDPR require any personal data that is collected or
 
 *In Swedish:* registrerad
 
-Any individual person who can be identified, directly or indirectly, via an identifier such as a name, an ID number, location data, or via factors specific to the person's physical, physiological, genetic, mental, economic, cultural or social identity.
+Any person who can be identified, directly or indirectly, via an identifier such as a name, an ID number, location data, or via factors specific to the person's physical, physiological, genetic, mental, economic, cultural or social identity.
```

Comment:
```suggestion
Any person that can be identified, directly or indirectly, via an identifier. Identifiers may include a name, an ID number, location data, or factors specific to the person's physical, physiological, genetic, mental, economic, cultural, or social identity.
```

Reply:
I do not really like to close a sentence and begin the next one with the same word (in this case identifier). But it can work as you have suggested!